pkg/volume/secret/secret.go

```diff
@@ -204,11 +204,11 @@ func (b *secretVolumeMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs
 	}
 
 	totalBytes := totalSecretBytes(secret)
-	klog.V(3).Infof("Received secret %v/%v containing (%v) pieces of data, %v total bytes",
-		b.pod.Namespace,
-		b.source.SecretName,
-		len(secret.Data),
-		totalBytes)
+	klog.V(3).Infof("Received secret containing pieces of data",
```

Comment:
```suggestion
	klog.V(3).InfoS("Received secret containing pieces of data",
```

Reply:
This has been fixed, waiting for your review with pleasure.